tests/acceptance/course-page/code-examples/vote-test.js

```
@@ -49,7 +49,7 @@
     await coursePage.codeExamplesTab.solutionCards[0].upvoteButton.hover();
 
     assertTooltipContent(assert, {
-      contentString: "Upvote this example and we'll show it more often.",
+      contentString: "Your feedback helps us surface better examples.",
```

ESLint

[error] Replace `"Your·feedback·helps·us·surface·better·examples."` with `'Your·feedback·helps·us·surface·better·examples.'`

(prettier/prettier)